pkgs/applications/networking/seaweedfs/default.nix

```
@@ -15,6 +20,11 @@ buildGoModule rec {
 
   subPackages = [ "weed" ];
 
+  passthru.tests.check-version = runCommand "weed-version" {} ''
```

```suggestion
  passthru.tests.check-version = runCommand "weed-version" { meta.timeout = 3; } ''
```
add timeout for hydra